components/02-molecules/pager/pager.twig

```diff
@@ -0,0 +1,79 @@
+{% set pager__base_class = 'pager' %}
```

Comment:
This file seems like it should be split into partial at the very least (see main feedback from the PR review). It's using 4 different types of markup with at least 4 levels of embedding said markup. If you decide to not extend the base menu for pager I would probably partial out this file to `pager-list.twig`, `pager-item.twig`, and `pager-link.twig`.

Reply:
I think I disagree... this is a very different component from a typical "menu". There are a lot of custom things this needs to support (`items.previous` (and next), `items.first` (and last), and then the "regular" `items.pages` - They're all mixed together in one array - and no other menu has those things.

This one also needs to pass icons into the content of specific items, which I think would be a pain to shoehorn into the base menu component.

There's also the fact that the "desktop" and "mobile" pagers are both here, but each are hidden at relevant screen sizes.

Also, the base menu components add a lot of extra stuff like menu levels - which will never apply to a pager.

On a side - but related - note, I wanted to talk with you about our implementation of the list component. I'm not sure the value is worth the complexity it adds. Let's zoom tomorrow, if you're free.